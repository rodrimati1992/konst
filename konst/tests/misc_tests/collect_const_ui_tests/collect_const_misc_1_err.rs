const _: [u8; 10] = {
    konst::iter::collect_const!{usize => 0usize..10}
};

const _: [usize; 9] = {
    konst::iter::collect_const!{usize => 0usize..10}
};

fn main(){}