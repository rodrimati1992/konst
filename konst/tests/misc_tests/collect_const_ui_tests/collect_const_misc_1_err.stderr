error[E0308]: mismatched types
 --> tests/misc_tests/collect_const_ui_tests/collect_const_misc_1_err.rs:2:5
  |
2 |     konst::iter::collect_const!{usize => 0usize..10}
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `[u8; 10]`, found `[usize; 10]`
  |
  = note: expected array `[u8; 10]`
             found array `[usize; 10]`
  = note: this error originates in the macro `konst::iter::collect_const` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0308]: mismatched types
 --> tests/misc_tests/collect_const_ui_tests/collect_const_misc_1_err.rs:6:5
  |
5 | const _: [usize; 9] = {
  |                  - help: consider specifying the actual array length: `10`
6 |     konst::iter::collect_const!{usize => 0usize..10}
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected an array with a fixed size of 9 elements, found one with 10 elements
  |
  = note: this error originates in the macro `konst::iter::collect_const` (in Nightly builds, run with -Z macro-backtrace for more info)
